python/py-pytest-html/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       pytest-html
+name                py-${real_name}
+version             2.0.1
+revision            0
+categories-append   devel
+
+license             Mozilla-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Plugin for generating HTML reports for pytest results
+
+long_description    pytest-html is a plugin for pytest that generates\
+                    a HTML report for the test results.
+
+homepage            https://pypi.python.org/pypi/${real_name}
+
+master_sites        pypi:p/${real_name}/
+distname            ${real_name}-${version}
+
+checksums           rmd160  884b4c79d2293f98e70ef6365fe1397a60bf2555 \
+                    sha256  933da7a5e71e5eace9e475441ed88a684f20f6198aa36516cb947ac05edd9921 \
+                    size    27905
+
+python.versions     37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools_scm
+
+    depends_run-append \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-pytest-metadata
+
+    post-destroot {
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${worksrcpath} README.rst LICENSE \
+            ${destroot}${docdir}
+    }
+
+    livecheck.type  none
+} else {
+    livecheck.type      regex
+    livecheck.url       https://pypi.python.org/pypi/${real_name}
+    livecheck.regex     "${real_name}/(\\d+(?:\\.\\d+)*(-p\\d+)?)"
+}
```

Comment:
remove the whole ```else``` block; the default PyPI livecheck is sufficient and will work

Reply:
Thanks done.